webrtc-stats.html

```diff
@@ -183,10 +183,11 @@
           Lifetime of stats objects
         </h3>
         <p>
-          The general rule is that stats objects, once created, exist for the lifetime of the
-          PeerConnection that contains them, even when the underlying object they report on is
-          stopped or deleted. This is important in order to report consistently on short-lived
-          objects and to be able to report totals over the lifetime of a PeerConnection.
+          The general rule is that stats objects are created at the same time as the underlying
```

Comment:
Is this covered by: "Conformance requirements phrased as algorithms or specific steps may be implemented in any manner, so long as the end result is equivalent"?